### PR TITLE
Fix passphrase prompt on PGP decrypt

### PR DESCRIPTION
**Pull request by pzduniak**
It's just a typo so I don't think we need tests for this. Thanks Patrick!

### CODE REVIEW

File: go/engine/scankeys.go [hunk 1]
```diff
@@ -314,7 +314,8 @@ func (s *ScanKeys) unlockByID(m libkb.MetaContext, id uint64) openpgp.EntityList
 			Reason:   unlockReason,
 			SecretUI: m.UIs().SecretUI,
 		}
-		unlocked, err := skb.PromptAndUnlock(m, parg, nil, s.me)
+		secretStore := libkb.NewSecretStore(m.G(), s.me.GetNormalizedName())
```
Comment (patrickxb) [inline go/engine/scankeys.go]: why this instead of `m.G().SecretStore()`?

**Comment by patrickxb** [comment]
There is a test for this that does look like it is checking that there was a passphrase prompt.  See `TestPGPDecryptWithSyncedKey`.  It might need to change.

File: go/engine/scankeys.go (hunk 1 above)
Comment (pzduniak) [reply go/engine/scankeys.go]: SecretStore returns SecretStoreLocked, which is a wrapper for SecretStoreAll. NewSecretStore takes m.G().SecretStore and returns their per-user store.

Comment (patrickxb) [reply go/engine/scankeys.go]: ok

**Pull request merged**